docs(upgrade): Prepares upgrade docs for v6 alpha. (#3885)
patternfly/patternfly-org: packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

import './get-started.css';
import { Divider } from '@patternfly/react-core';

## PatternFly 6 alpha

We're excited to announce that our alpha release of PatternFly 6 is now live. Here are the highlights that you should know about!

> ### Penta

Author: ```suggestion
### PatternFly 6 visual theme
```

> As you've probably already noticed on the alpha website: PatternFly has a new look. Our brand new visual theme is called "Penta". Its visual style is more modern and refined, so that you can create UIs that are more seamless and lightweight.

Author: ```suggestion
As you've probably already noticed on the alpha website: PatternFly 6 has a new look. Our brand new visual theme is more modern and refined, so that you can create UIs that are more seamless and lightweight.
```

All of our components have a new look to match. As you use the alpha website, take a look at our component examples, demos, and patterns to explore our redesign.

### Design tokens 

> In order to support Penta, and any future visual theming capabilities, we have implemented a design token system for PatternFly. For more details and instructions on how to use tokens, you can refer to our new [design token documentation.](/tokens/about-tokens)

Author: ```suggestion
In order to support PatternFly 6, and any future visual theming capabilities, we have implemented a design token system for PatternFly. For more details and instructions on how to use tokens, you can refer to our new [design token documentation.](/tokens/about-tokens)
```
